backend/.env.example

```
@@ -12,3 +12,13 @@ CHROMA_DB_PATH=./chroma_db
 # Supabase PostgreSQL with transaction pooler (port 6543)
 # Format: postgresql+asyncpg://postgres.[PROJECT_REF]:[PASSWORD]@aws-0-[REGION].pooler.supabase.com:6543/postgres?prepared_statement_cache_size=0
 DATABASE_URL=postgresql+asyncpg://postgres.your-project-ref:your-password@aws-0-eu-west-1.pooler.supabase.com:6543/postgres?prepared_statement_cache_size=0
+
+# Security
+CRON_SECRET_KEY=dev-cron-secret-key
+SECRET_KEY=your-secret-key-here-change-in-production
+BACKEND_API_KEY=your-backend-api-key-here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BACKEND_API_KEY key should go before the CRON_SECRET_KEY key

(UnorderedKey)